### DISCUSSION TITLE
Modal and Workspace issues

### DISCUSSION THREAD

**Discussion by nickm324** [Lissy93/dashy#594] [Help and Support] [answered]
Most of the links I try and configure with Modal or Workspace fail with "refused to connect", but all these same links work fine opening in the SAME or NEWTAB. 

Is there a reason like iframe issues, is there something that can be configured to get them working in Modal or Workspace?

**Comment by lissy93** [comment, marked as answer]
Yup, it's just CORS, there's some info about fixing it in the docs and [troubleshooting guide](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md) (first section at the top)
